internal/pkg/scaffold/go_mod.go

```diff
@@ -61,7 +60,7 @@ replace (
 replace (
 	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.31.1
 	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.2.0-beta.3
-	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde
+	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.2.0-beta.3
```

Comment:
Do Go operator projects use controller-tools? Or can this be removed?

Ditto for `Gopkg.toml`